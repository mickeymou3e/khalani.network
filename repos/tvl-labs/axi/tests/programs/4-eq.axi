../../tutorial/4-eq.axi