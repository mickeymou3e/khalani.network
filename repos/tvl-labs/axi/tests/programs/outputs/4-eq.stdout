Result: Nothing
Result: Nothing
Result: Nothing
Result: Nothing
Result: Proved(forall N. forall N. forall N. forall N. if(=[@3:N, @1:N], if(=[@2:N, @0:N], =[g[@3:N, @2:N]):N, g[@1:N, @0:N]):N])))
Result: Proved(forall N. forall N. forall N. if(=[@2:N, @1:N], if(=[@1:N, @0:N], =[@2:N, @0:N])))
Result: Proved(forall N. =[@0:N, @0:N])
Result: Proved(forall N. forall N. forall N. forall N. if(=[@3:N, @1:N], if(=[@2:N, @0:N], =[g[@1:N, @0:N]):N, g[@3:N, @2:N]):N])))
Result: Proved(forall N. forall N. forall N. if(=[g[@2:N, @1:N]):N, f[@1:N]):N], if(=[@1:N, h[@0:N]):N], =[g[@2:N, @1:N]):N, f[h[@0:N]):N]):N])))
